core/domain/collection_domain.py

```diff
@@ -376,15 +376,30 @@ def __init__(self, skill_id, name, question_ids):
         self.question_ids = question_ids
 
     def to_dict(self):
-        # Note to developers: ensure this matches the frontend in
-        # CollectionSkillObjectFactory
+        """Returns the dict of CollectionSkill object.
+
+        Note to developers: Ensure this matches the frontend in
+        CollectionSkillObjectFactory.
+
+        Returns:
+            dict. The dict of CollectionSkill object.
```

Comment:
dict. A dict version of the CollectionSkill object.

Reply:
Done